routing/index_router.cpp

```diff
@@ -374,6 +374,59 @@ IRouter::ResultCode IndexRouter::CalculateRoute(Checkpoints const & checkpoints,
   }
 }
 
+bool IndexRouter::FindBestSegment(m2::PointD const & point, m2::PointD const & direction,
+                                  bool isOutgoing, WorldGraph & worldGraph, Segment & bestSegment,
+                                  bool & bestSegmentIsAlmostCodirectional) const
+{
+  auto const file = platform::CountryFile(m_countryFileFn(point));
```

Comment:
What if multiple mwms cover the |point|?

Reply:
The next question is what if no mwm cover the |point|. In these cases in generator we may get: MYTHROW(RoutingException, ("Can't get mwm handle for", file)); I guess.

|may| because if multiple mwms cover the |point| |m_countryFileFn| returns an arbitrary of them.

I'll make try/catch for RoutingException in case of a call IndexRouter::FindBestSegment() in generator.

BTW. While routing we may face such problem if we set start/finish/via point near a border of downloaded mwm. There's a try/catch in that case.